api/api/services/sendgrid_mail_global_send.py

```diff
@@ -0,0 +1,172 @@
+import os
```

Comment:
don't copy everything to a new file to do your implementation. Instead, merge your work with the existing one